contracts/bridges/HomeMediator.sol

```diff
@@ -104,13 +104,13 @@ contract HomeMediator {
     ///        - payload as bytes, with the length equal to the specified payload length.
     function processMessageFromForeign(bytes memory data) external {
         // Check for the AMB Mediator address
-        if (msg.sender != AMBMediator) {
-            revert AMBMediatorOnly(msg.sender, AMBMediator);
+        if (msg.sender != AMBContractProxyHome) {
```

Comment:
Comment line 97, maybe would be better to replace "AMB Mediator contract" with  "AMBContractProxyHome"